components/treelist/editing/overview.md

```diff
@@ -380,6 +380,16 @@ There are a few considerations to keep in mind with the CUD operations of the tr
 @* Applies to the other input type components as well *@
 ````
 
+* If you want to perform other data operations while the component is in Edit mode (applicable for [InCell]({%slug treelist-editing-incell%}) and [Inline]({%slug treelist-editing-inline%}) editing) the following behavior will occur:
```

Comment:
Consider a _contentTemplate for this repeating content.

Reply:
The links in this exact paragraph are different (one points to the corresponding articles for grid and the other - for treelist). However, I may use a template for the rest of the content.

Reply:
You can consider using relative links rather than slugs to put this in a template.